### PR TITLE
Create downloads page

### LINKED ISSUE #9: [Feature Request]: Need to make the download for windows page.
image

Need to create the page when the user clicks the Download for Windows button.

### PR DESCRIPTION
**Pull request by Vaibhav1808**
## Description
Using HTML CSS and JavaScript, adding a download for windows page in index.html 

## Issue Resolved

#9 

## Screenshots 

image

## Checklist

Please make sure to review the following before submitting your PR: 

- [x] I have read the [contribution guidelines](https://github.com/PEC-CSS/MovieDroid/blob/main/CONTRIBUTING.md).
- [x] I have read the code of conduct.
- [x] I have reviewed my submission in detail.

### CODE REVIEW

File: pages/download.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+      <!-- search bar -->
+      <form class="search">
+        <input type="text/javascript" placeholder="Search entire site..." autocomplete="off" />
+      </form>
+    </header>
+
+    <div class="inner">
+        <div id="content-wrapper">
+          <button class="sidebar-btn"></button>
```
Comment (Manobal-Singh-Bagady) [inline pages/download.html]: remove this button for now as the sidebar functionality cannot be implemented.

**Review by Manobal-Singh-Bagady — CHANGES_REQUESTED** [review]
Your Preview:
image

Intended Sidebar:
image



Change the following things:
- [x]  Color of the headings in the sidebar
- [x]  Color of the subheadings in the sidebar
- [x]  Remove the button in the sidebar.

Other than that, everything looks good. Good Work @Vaibhav1808

**Comment by Vaibhav1808** [comment]
Implemented suggested changes

File: pages/download.html (hunk 1 above)
Comment (Vaibhav1808) [reply pages/download.html]: Done

**Review by Manobal-Singh-Bagady — APPROVED** [review]
GW @Vaibhav1808, LGTM

**Review by Manobal-Singh-Bagady — COMMENTED** [review]
Merging this PR

**Pull request merged**